src/components/media_manager/src/socket_streamer_adapter.cc

```diff
@@ -70,6 +70,8 @@ SocketStreamerAdapter::SocketStreamer::SocketStreamer(
     ip_(ip),
     port_(port),
     header_(header),
+    server_socket_(),
```

Comment:
I understood you. At the moment there is no issues. But in the long term, during some refactoring, someone can add another member and break the init order this way. That is why it is considered that explicit init is better.